gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/baseline.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.hypertrace.gateway.service.v1.baseline";
+
+package org.hypertrace.gateway.service.v1.health;
+
+import "org/hypertrace/gateway/service/v1/gateway_query.proto";
+
+message BaselineEntity {
+  string id = 1;
+  string entity_type = 2;
+  map<string, org.hypertrace.gateway.service.v1.common.Value> attribute = 4;
```

Comment:
Attributes are not needed on a baseline entity

Reply:
done